common/list.h

```diff
@@ -131,48 +131,49 @@ class List {
 				i = i->_next;
 	}
 
-	/** Inserts element at the start of the list. */
+	/** Insert an @p element at the start of the list. */
 	void push_front(const t_T &element) {
 		insert(_anchor._next, element);
 	}
 
-	/** Appends element to the end of the list. */
+	/** Append an @p element to the end of the list. */
 	void push_back(const t_T &element) {
 		insert(&_anchor, element);
 	}
 
-	/** Removes the first element of the list. */
+	/** Remove the first element of the list. */
 	void pop_front() {
 		assert(!empty());
 		erase(_anchor._next);
 	}
 
-	/** Removes the last element of the list. */
+	/** Remove the last element of the list. */
 	void pop_back() {
 		assert(!empty());
 		erase(_anchor._prev);
 	}
 
-	/** Returns a reference to the first element of the list. */
+	/** Return a reference to the first element of the list. */
```

Comment:
If 'returning' is the only thing that the function does, I think it's better to keep it as a ``@brief`` (we are using autobrief so all these descriptions are essentially a ``@brief`` element). Honestly, I'm not even sure if it would be possible to have a ``@return`` without a brief first. And duplicating the same thing in both brief and return does not make much sense.

That said, ``@return`` or ``@retval`` is of course a great thing to have in more complex functions that can return various values.

Reply:
I am not familiar with `@brief` (we decided to turn it off at work) and forgot it exists. But I think that what you wrote makes sense.